src/payment/strategies/amazon-maxo/amazon-maxo.mock.ts

```diff
@@ -0,0 +1,58 @@
+import PaymentMethod from '../../payment-method';
+
+import { AmazonMaxoButtonParams, AmazonMaxoCheckoutLanguage, AmazonMaxoLedgerCurrency, AmazonMaxoPlacement, AmazonMaxoSDK } from './amazon-maxo';
```

Comment:
you can split this import in several lines to keep readability 🍹

Reply:
@FranciscoOnt it's not possible due to:
https://github.com/bigcommerce/checkout-sdk-js/blob/6931d3e46f1e3e47abb36dc0c1b3f23d09b85fab/.eslintrc.json#L45